### ISSUE TITLE
Feature request: SSIM loss

### ISSUE THREAD

**Issue by John1231983** [junyanz/pytorch-CycleGAN-and-pix2pix#624]
For tensorflow, [someone ](https://github.com/Tandon-A/CycleGAN_ssim)has success using structural similarity (SSIM) loss for cycle consistency loss. It shows the very impressive results
![Sim](https://user-images.githubusercontent.com/24875971/56461219-e9232580-637c-11e9-9f40-54ed17d412a5.JPG)

Do you have the plan to consider the loss in the project? Thanks

**Comment by junyanz** [comment]
SSIM looks great. We don't have a plan to add it. But feel free to do a PR request.

**Comment by arrrrr3186** [comment]
HI @John1231983,
did you implemented SSIM Loss function ?
